contracts/Unibright.sol

```diff
@@ -218,10 +227,14 @@ contract UBTSplitter is Context, Ownable {
         payees.push(revenueAddress);
         validatorStakingAddress[revenueAddress] = stakingAddress;
```

Comment:
i think there is big chunk of same code in addPayee and updatePayee maybe good idea is to move it to small private function?